dip-template/runtimes/dip-consumer/src/lib.rs

```diff
@@ -358,6 +355,8 @@ impl pallet_aura::Config for Runtime {
 	type AuthorityId = AuraId;
 	type DisabledValidators = ();
 	type MaxAuthorities = ConstU32<100_000>;
+	#[cfg(feature = "experimental")]
```

Comment:
This feature does not exist.

Reply:
[952013d](https://github.com/KILTprotocol/kilt-node/pull/627/commits/952013d76a9cb7fcc3e5c21d9e1b9f4679499c49)